parsel/selector.py

```diff
@@ -217,13 +220,20 @@ def get(self, default: None = None) -> Optional[str]:
     def get(self, default: str) -> str:
         pass
 
-    def get(self, default: Optional[str] = None) -> Optional[str]:
+    @typing.overload
+    def get(self, strip: bool) -> str:
+        pass
```

Comment:
Why is this necessary? (I am not saying it is not)

Reply:
I don't think it is, I just saw the overloads above this line and thought it was required

```
@typing.overload
def get(self, default: None = None) -> Optional[str]:
    pass

@typing.overload
def get(self, default: str) -> str:
    pass
```

I removed it on the last push. I don't understand the whole overload completely, but I feel those 2 are also unnecessary?

Reply:
I think it is related to typing, when parameter types affect output type, but I do not think it applies here.

Reply:
As you added a new argument to the method you need to add them to all overloads too.

Reply:
And those two overloads are needed because the method either can return None if default wasn't passed or it returns the default instead, so this describes the typing info more clearly.